tests/Component/Transport/ConnectorTest.php

```diff
@@ -113,10 +111,9 @@ public function testSendGuzzleError()
         $response = new Response(404);
         $this->mock->append($response);
 
-        $this->setExpectedException('GuzzleHttp\Exception\ClientException');
-
         $request = $this->connector->createRequest('http://somewhere/path', 'POST');
```

Comment:
Use constant from Method class instead of string

Reply:
Replaced all the text methods in all test files.